README.md

```diff
@@ -59,6 +59,8 @@ The AWS IoT Device SDK for Embedded C (C-SDK) is a collection of C source files
 
 For the latest release of C-SDK, please see the section for [Releases](#releases).
 
+**C-SDK includes libraries that are part of the [FreeRTOS 202012.00 LTS](https://github.com/FreeRTOS/FreeRTOS-LTS/tree/202012.00_LTS) release. Learn more about the FreeRTOS 202012.00 LTS libraries at https://freertos.org/lts-libraries.html.**
```

Comment:
pls confirm https://github.com/FreeRTOS/FreeRTOS-LTS/tree/202012.00_LTS is the right link with Joe.

Reply:
the right link is https://github.com/FreeRTOS/FreeRTOS-LTS/tree/202012.00-LTS